### ISSUE TITLE
New version gives an error when creating a client instance.

### ISSUE THREAD

**Issue by SykkunoSimp** [ading2210/poe-api#86]
Everything still seem to work fine even with that error. I got the following output with the use_3rd_party_bot example.

Exception in thread Thread-1 (get_bot_thread):
Traceback (most recent call last):
Exception in thread Thread-2 (get_bot_thread):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
Traceback (most recent call last):
Exception in thread Thread-4 (get_bot_thread):
Exception in thread Thread-5 (get_bot_thread):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
Exception in thread Thread-3 (get_bot_thread):
Traceback (most recent call last):
Exception in thread Thread-6 (get_bot_thread):
Exception in thread Thread-8 (get_bot_thread):
Traceback (most recent call last):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
    self.run()
Traceback (most recent call last):
Exception in thread Thread-7 (get_bot_thread):
Traceback (most recent call last):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
Traceback (most recent call last):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
Traceback (most recent call last):
    self.run()
    self.run()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
    self.run()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
Exception in thread Thread-9 (get_bot_thread):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
    self._target(*self._args, **self._kwargs)
    self.run()
    self.run()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
    chat_data = self.get_bot(bot["displayName"])
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    self.run()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    self.run()
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
Traceback (most recent call last):
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
    self._target(*self._args, **self._kwargs)
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    self.run()
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 975, in run
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
    self._target(*self._args, **self._kwargs)
  File "C:\Users\HP\AppData\Local\Programs\Python\Python311\Lib\site-packages\poe.py", line 204, in get_bot_thread
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
    chat_data = self.get_bot(bot["displayName"])
                             ~~~^^^^^^^^^^^^^^^
KeyError: 'displayName'
Bot name: JapaneseTutor
こんにちは。"Hi there"は日本語で「こんにちは」と言います。直訳すると「そこにこんにちは」となりますが、実際には「やあ」「こんにちは」という挨拶の意味を持ちます。

**Comment by TheLime1** [comment]
me too it messed up my programm, even [the example](https://github.com/ading2210/poe-api/blob/main/examples/send_message.py) dosnt work anymore

**Comment by ading2210** [comment]
This was due to a change with Poe, not a fault with this library. They changed the `availableBots` key to `viewerBotList` in the viewer data.

**Comment by ading2210** [comment]
This has been fixed in v0.4.2.
